### ISSUE TITLE
mechanism to see method signature of completed function

### ISSUE THREAD

**Issue by bryanwb** [lihaoyi/Ammonite#26]
@lihaoyi tks for the awesome ammonite-repl! I am very excited to use it

There is one feature that I do miss. In the regular scala repl, i can hit <TAB> after a complete method name to see a method signature(s)

for example

```shell
scala> foo.replace<TAB>
   def replace(Char, Char): String   def replace(CharSequence, CharSequence): String
```

it would be sweet to get the same kind of behavior in ammonite-repl

**Comment by lihaoyi** [comment]
Yeah that looks doable...

**Comment by lihaoyi** [comment]
Fixed in c825638838958d6168db2105ca9748d6e6247ef4

**Comment by bryanwb** [comment]
@lihaoyi awesome! tks
